### PR TITLE
OKTA-296724: Dev docs tree nav - scroll to the right section

### PR DESCRIPTION
**Pull request by oleksandrpravosudko-okta**
## Description:
- **What's changed?**
Highlighted tree nav section becomes visible when navigating by URL, for example [docs/reference/api/user-types](https://developer.okta.com/docs/reference/api/user-types/)

- **Is this PR related to a Monolith release?**
No

UPD: screen captures have been replaced to reflect latest code changes

Before fix:
ezgif com-gif-maker (1)

After fix:
ezgif com-gif-maker

### Resolves:

* [OKTA-296724](https://oktainc.atlassian.net/browse/OKTA-296724)

### CODE REVIEW

File: packages/@okta/vuepress-theme-prose/components/SidebarItem.vue [hunk 1]
```diff
@@ -10,7 +10,7 @@
             </svg>
 
             <div v-if="link.path">
-                <router-link :to="link.path" @click="setData" :class="{'router-link-active': link.imActive, 'router-link-exact-active': link.imActive}">{{link.title}}</router-link>
+                <router-link :to="link.path" @click="setData" :class="{'tree-nav-link': true, 'router-link-active': link.imActive, 'router-link-exact-active': link.imActive}">{{link.title}}</router-link>
```
Comment (AndriiDovhopolov-okta) [inline packages/@okta/vuepress-theme-prose/components/SidebarItem.vue:13]: Do we need both "router-link-active" and "router-link-exact-active"? I mean, both of classes are dependent on link.imActive property

**Comment by alexdahl-okta** [comment]
Demo looks good to me, thanks!

File: packages/@okta/vuepress-theme-prose/components/SidebarItem.vue (hunk 1 above)
Comment (oleksandrpravosudko-okta) [reply packages/@okta/vuepress-theme-prose/components/SidebarItem.vue:13]: it looks like these classes were used to toggle font weight under [`.overview`](https://github.com/okta/okta-developer-docs/blob/31f38a4ee918658663ddcce18913891f4231b1f4/packages/%40okta/vuepress-theme-prose/assets/css/components/_sidebar.scss#L50) section (though markup with `.overview` class doesn't seem to be around anymore)
I agree we can go with one class now, however I suggest we cover it in OKTA-320822(i.e. remove bold font-weight for non-ancestors by dropping `.router-link-active`).

Comment (AndriiDovhopolov-okta) [reply packages/@okta/vuepress-theme-prose/components/SidebarItem.vue:13]: Got it, thanks. Will merge this changes to me.

**Review by bryanapellanes-okta — COMMENTED** [review]
This looks good to me 👍 
I defer approval to someone more closely familiar with the related area.

**Review by bretterer — APPROVED** [review]

**Pull request merged**